tests/autoyast/verify_lvm_partitions.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Verify lvm partitions after autoyast installation
+# Maintainer: qa-sle-yast <qa-sle-yast@suse.com>
+
+use strict;
+use warnings;
+use parent 'installbasetest';
+use testapi;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my $test_data = get_test_suite_data();
+
+    my $vg = $test_data->{profile}->{partitioning}->[1]->{drive}->{partitions}->[0]->{partition}->{lvm_group};
```

Comment:
When it goes to such a long line in getting data, especially when referring to some array indexes, I guess it should tell us that we are doing something wrong.

Seems like the drives in the yaml schedule are different entities. Just by fields that they have: first drive includes partitions with `lv_name` and `mount` fields, when the second one introduced partitions that do not have such fields. I would introduce those drives as separate data units.

Reply:
Well, the test data is formatted this way because of  [validate_autoyast_profile](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7043fd239e3a28cf5dfd9590cac5781a5a69e731/lib/autoyast.pm#L206), we need to "mimic" the xml structure, not for this specific module but for verify_cloned_profile.

Reply:
as per several comments, I believe is better not used at all the test_data used for validated the cloned profile.
we need to mimic that only to make happy the parser in that module, but for the rest of the test IMO doesn't make sense.

Reply:
Ok, I'll do that.